calico_containers/calicoctl.py

```diff
@@ -1648,275 +84,152 @@ def permission_denied_error(conn_error):
-                ip_pool_add(arguments["<CIDR>"],
-                            ip_version,
-                            arguments["--ipip"],
-                            arguments["--nat-outgoing"])
-            elif arguments["remove"]:
-                ip_pool_remove(arguments["<CIDR>"], ip_version)
-            elif arguments["show"]:
-                if not ip_version:
-                    ip_pool_show("v4")
-                    ip_pool_show("v6")
-                else:
-                    ip_pool_show(ip_version)
-        elif arguments["bgppeer"]:
-            if arguments["add"]:
-                bgppeer_add(arguments["<PEER_IP>"], ip_version,
-                            arguments["<AS_NUM>"])
-            elif arguments["remove"]:
-                bgppeer_remove(arguments["<PEER_IP>"], ip_version)
-            elif arguments["show"]:
-                if not ip_version:
-                    bgppeer_show("v4")
-                    bgppeer_show("v6")
-                else:
-                    bgppeer_show(ip_version)
-        elif arguments["container"]:
-            if arguments["endpoint-id"]:
-                container_endpoint_id_show(arguments["<CONTAINER>"])
-            elif arguments["ip"]:
-                if arguments["add"]:
-                    container_ip_add(arguments["<CONTAINER>"],
-                                     arguments["<IP>"],
-                                     ip_version,
-                                     arguments["--interface"])
-                elif arguments["remove"]:
-                    container_ip_remove(arguments["<CONTAINER>"],
-                                        arguments["<IP>"],
-                                        ip_version,
-                                        arguments["--interface"])
-                else:
-                    if arguments["add"]:
-                        container_add(arguments["<CONTAINER>"],
-                                      arguments["<IP>"],
-                                      arguments["--interface"])
-                    if arguments["remove"]:
-                        container_remove(arguments["<CONTAINER>"])
-            else:
-                if arguments["add"]:
-                    container_add(arguments["<CONTAINER>"],
-                                  arguments["<IP>"],
-                                  arguments["--interface"])
-                if arguments["remove"]:
-                    container_remove(arguments["<CONTAINER>"])
-        elif arguments["bgp-node-mesh"]:
-            if arguments["on"] or arguments["off"]:
-                set_bgp_node_mesh(arguments["on"])
-            else:
-                show_bgp_node_mesh()
-        elif arguments["default-node-as"]:
-            if arguments["<AS_NUM>"]:
-                set_default_node_as(arguments["<AS_NUM>"])
-            else:
-                show_default_node_as()
+    # TODO: come up with a better way to default to --help when no opts provided
```

Comment:
I think this approach is fine, so remove the TODO but leave a comment explaining why you're doing this.